app.test.js

```diff
@@ -48,4 +49,40 @@ describe("GET /api", () => {
             expect(body).toHaveProperty("response")
         })
     })
+})
+
+
+describe.only("GET /api/articles/:article_id", () => {
+    test("returns an article object for the article for that id path", () => {
+        return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then(({body}) => {
+            const desiredArticle = body.response
+            
+            expect(typeof desiredArticle).toBe("object")
+            expect(Object.keys(desiredArticle)).toHaveLength(8)
+
+            expect(desiredArticle).toHaveProperty("author")
+            expect(desiredArticle).toHaveProperty("title")
+            expect(desiredArticle).toHaveProperty("article_id")
+            expect(desiredArticle).toHaveProperty("body")
```

Comment:
This does work.
I would suggest you look into some other Jest Matchers which would be better for this test and future tests to come.
There is two in particular that will be useful to you this week. ObjectContaining and ToMatchObject.